packages/lib/tsconfig.json

```diff
@@ -15,7 +15,7 @@
         "noImplicitAny": false,
         "strictPropertyInitialization": false,
         "suppressImplicitAnyIndexErrors": true,
-        "stripInternal": true,
+        "stripInternal": false,
```

Comment:
Since false is the default value, I would recommend simply removing the property. This also gives you a better overview if only properties that do not have the default value can be found in the tsconfig file.